WalletSdk/src/main/java/com/spruceid/walletandroidsdk/BleCentral.kt

```diff
@@ -0,0 +1,156 @@
+package com.spruceid.walletandroidsdk
```

Comment:
To be consistent with the other libraries I think it should be `com.spruceid.wallet.sdk:walletsdk`